artichoke-backend/build.rs

```diff
@@ -195,14 +196,16 @@ fn main() {
         // "win32ole", native code, not requirable on all platforms
         "yaml",
         "zlib",
-    ] {
+    ];
+
+    packages.par_iter().for_each(|package| {
         let sources = Build::get_package_files(package)
             .trim()
             .split("\n")
             .filter(|s| !s.is_empty())
             .map(String::from)
             .collect::<Vec<_>>();
```

Comment:
I don't think we need to collect into a `Vec<_>` anymore since we're consuming the iterator directly with rayon.

Reply:
This might be possible with `ParallelBridge`, but the docs say this is less efficient than using a `Vec`. Our list is small so it makes sense to pay the cost to collect eagerly.